src/fides/api/schemas/privacy_request.py

```diff
@@ -65,7 +64,7 @@ class ConsentReport(Consent):
     """Schema for reporting Consent requests."""
 
     id: str
-    identity: IdentityBase
+    identity: Identity
```

Comment:
Does this also surface `phone_number` / `email` if provided (via privacy center requests, specifically)? Do we want to, if so?

Reply:
This `ConsentReport` schema is used only for consent requests — it will surface that information if associated with the consent requests, which it _needs_ to do to prove compliance.